modules/social_features/social_profile/modules/social_profile_fields/social_profile_fields.install

```diff
@@ -20,6 +20,7 @@ function social_profile_fields_install() {
   _social_profile_fields_update_search_index();
   _social_profile_fields_update_search_index('social_all');
   _social_profile_fields_nationalities();
+  _social_profile_fields_grunt_nationalities_permissions();
```

Comment:
@Macsym2018 in this case better to not create extra function, but grant permissions directly in hook_install:

```
$permissions = [
    'view terms in nationality',
    'view term names in nationality',
    'select terms in nationality',
  ];
  user_role_grant_permissions('sitemanager', $permissions);
```

Reply:
@ribel Done